content/en/docs/topics/permissions_sql_storage_backend.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Permissions management for SQL storage backend"
+description: "Get to know how to properly setup permissions when using SQL storage backend."
+aliases: ["/docs/permissions_sql_storage_backend/"]
+---
+
+This document aims to provide guidance to Helm 3 users for setting up and managing permissions properly when using the SQL storage backend.
+
+## Introduction
+
+To handle permissions, Helm leverages the RBAC feature of Kubernetes. When using the SQL storage backend, Kubernetes' roles can't be used to determine whether or not an user can access a given resource. We'll here discuss how to properly create and manage these permissions.
+
+## Initialization
+
+The first time the Helm CLI will make connect to your database, the client will make sure that it was previously initialized. If it was not, it will take care of running the migration scripts. This initialization requires admin privileges on the public schema, or at least to be able to:
```

Comment:
I am unsure what you mean here. What are these migration scripts that you refer to? Are you talking about the initialization of the DB and the creation of roles? Who or what runs this initialization?

Reply:
https://github.com/helm/helm/blob/8e5e6caee2b487534858540c2944668512e316e7/pkg/storage/driver/sql.go#L85-L158

Reply:
If it was not, it will take care of running the migration scripts. -> If it is not, it will take care of the necessary setup automatically.